lib/msal-electron-proof-of-concept/src/appConfig/clientApplicationBase.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ApplicationConfiguration, buildApplicationConfiguration } from './applicationConfiguration';
+
+/**
+ * ClientApplicationBase abstract class
+ *
+ * Abstract class that provides a base for Public and Private Application classes.
+ */
+export abstract class ClientApplicationBase {
+    private _config: ApplicationConfiguration;
+
+    constructor(applicationConfiguration: ApplicationConfiguration) {
+        this._config = buildApplicationConfiguration(applicationConfiguration);
```

Comment:
>this [](start = 8, length = 4)

nit: do you need "this" prefixed here?

Reply:
I believe it is required even if the property doesn't the same name as the parameter, but if not, it's still convention (I haven't seen a constructor without `this` yet). In JS, the `this` keyword can get weird.